.github/workflows/fw_icu4c_ci.yml

```diff
@@ -32,6 +31,38 @@ jobs:
       uses: microsoft/setup-msbuild@v1.1
       if: ${{ matrix.os == 'windows-latest' }}
 
-    - name: Build
-      run: msbuild nugetpackage/build/icu4c.proj
-      working-directory: icu4c
\ No newline at end of file
+    - name: Build ICU 32 bit
+      working-directory: icu4c
+      run: msbuild source/allinone/allinone.sln /p:Configuration=Release /p:Platform=Win32 /p:SkipUWP=true
+      if: ${{ matrix.os == 'windows-latest' }}
+
+    - name: Prepare data for SIL tests
+      working-directory: icu4c
+      run: msbuild nugetpackage/build/icu4c.proj /t:PrepareTestData
+
+    - name: Test ICU 32 bit
+      run: source/allinone/icucheck.bat x86 Release
+      working-directory: icu4c
```

Comment:
The GH action fails in this step. Although, looking at the build log it's not very clear to my why. The tests seem to pass and yet the end result is failed? 

Ah, I see. The `iotest` test suite failed.

Reply:
Yes, when I changed the runtime that test suite started failing. :-(